### PR TITLE
added various missing release notes

### PR DESCRIPTION
**Pull request by BeateRixen**

### CODE REVIEW

File: content/release-10-16-0/platform-10-16-0-bundle/10-16-0-159.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Authentication</td>
+<td>The performance of the first user requests sent after node restart has been improved. The issue was causing problems in rare cases where the number of all global roles in a tenant was close to but not exceeding 100, and many thousands of devices were concurrently trying to authorize MQTT connections on the restarted node.</td>
```
Comment (rssag) [inline content/release-10-16-0/platform-10-16-0-bundle/10-16-0-159.md:25]: this talks about an improvement and then proceeds to talk about an "issue", which is a strange continuation. Maybe rephrase to "Before, there were problems in rare cases..."

File: content/release-10-17-0/devices-10-17-0.md [hunk 2]
```diff
@@ -197,6 +218,20 @@ mandatory <b>Software type</b> field is filled.</td>
...
+<tr>
+<td>LWM2M</td>
+<td>In certain cases the migration of LWM2M devices registered on versions before 10.15 reported "Unable to process device" for certain devices. This happened due to adjustments to the device configuration, for example when editing the registration lifetime. With this fix, the migration logic robustness has been improved.</td>
+<td>10.17.0.47</td>
+<td>DM-2022</td>
+</tr>
+
+<tr>
+<td>LWM2M</td>
+<td>In the older Leshan demo client, the device registration requests for the Cumulocity IoT LWM2M demo device and probably also some other devices could contain spaces between the core link separators. This was successfully handled by the LWM2M agent 10.14 and below but caused the LWM2M agent 10.15 and above to refuse these device requests. Now, the LWM2M server is more liberal and accepts these requests again. The older device registration requests can be processed again.</td>
```
Comment (rssag) [inline content/release-10-17-0/devices-10-17-0.md:230]: see above

File: content/release-10-18-0/devices-10-18-0.md [hunk 3]
```diff
@@ -80,6 +94,20 @@ Fix Version</th>
...
+<tr>
+<td>LWM2M</td>
+<td>In certain cases the migration of LWM2M devices registered on versions before 10.15 reported "Unable to process device" for certain devices. This happened due to adjustments to the device configuration, for example when editing the registration lifetime. With this fix, the migration logic robustness has been improved.</td>
+<td>10.17.107.0</td>
+<td>DM-2022</td>
+</tr>
+
+<tr>
+<td>LWM2M</td>
+<td>In the older Leshan demo client, the device registration requests for the Cumulocity IoT LWM2M demo device and probably also some other devices could contain spaces between the core link separators. This was successfully handled by the LWM2M agent 10.14 and below but caused the LWM2M agent 10.15 and above to refuse these device requests. Now, the LWM2M server is more liberal and accepts these requests again. The older device registration requests can be processed again.</td>
```
Comment (rssag) [inline content/release-10-18-0/devices-10-18-0.md:106]: see above

File: content/release-10-16-0/devices-10-16-0-bundle/10-16-0-159.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+<tr>
+<td>Device Management</td>
+<td>On the <b>Software</b> tab in the device details, the <b>Reload</b> button now correctly updates the <b>Installed software</b> list.</td>
+<td>DM-2005</td>
+</tr>
+
+<tr>
+<td>LWM2M</td>
+<td>In the older Leshan demo client, the device registration requests for the Cumulocity IoT LWM2M demo device and probably also some other devices could contain spaces between the core link separators. This was successfully handled by the LWM2M agent 10.14 and below but caused the LWM2M agent 10.15 and above to refuse these device requests. Now, the LWM2M server is more liberal and accepts these requests again. The older device registration requests can be processed again.</td>
```
Comment (BeateRixen) [inline content/release-10-16-0/devices-10-16-0-bundle/10-16-0-159.md:49]:
```suggestion
<td>In the older Leshan demo client, the device registration requests for the Cumulocity IoT LWM2M demo device and possibly also some other devices could contain spaces between the core link separators. This was successfully handled by the LWM2M agent 10.14 and below but caused the LWM2M agent 10.15 and above to reject these device requests. Now, the LWM2M server accepts these requests again. The older device registration requests can be processed again.</td>
```

**Review by rssag — APPROVED** [review]
